ci-operator/step-registry/proxy/config-remove/proxy-config-remove-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: proxy-config-remove
+  from_image:
+    namespace: origin
+    name: centos
+    tag: '8'
```

Comment:
CentOS 8 reach End Of Life (EOL) on December 31st, 2021. Can we switch to cli, or upi-installer, or another image that's part of OCP ?

Reply:
Yup, Done. Switched to `upi-installer`.